File: note_taking_app.js
const fs = require('fs');
const path = require('path');

function createNote() {
    const title = prompt("Enter the title of the note: ");
    const content = prompt("Enter the content of the note: ");
    fs.writeFileSync(`${title}.txt`, content);
    console.log("Note created successfully.");
}

function viewNote() {
    const title = prompt("Enter the title of the note to view: ");
    if (fs.existsSync(`${title}.txt`)) {
        const content = fs.readFileSync(`${title}.txt`, 'utf8');
        console.log("Note content:");
        console.log(content);
    } else {
        console.log("Note not found.");
    }
}

function deleteNote() {
    const title = prompt("Enter the title of the note to delete: ");
    if (fs.existsSync(`${title}.txt`)) {
        fs.unlinkSync(`${title}.txt`);
        console.log("Note deleted successfully.");
    } else {
        console.log("Note not found.");
    }
}

function main() {
    while (true) {
        console.log("1. Create a note");
        console.log("2. View a note");
        console.log("3. Delete a note");
        console.log("4. Exit");
        const choice = prompt("Enter your choice: ");
        if (choice === "1") {
            createNote();
        } else if (choice === "2") {
            viewNote();
        } else if (choice === "3") {
            deleteNote();
        } else if (choice === "4") {
            break;
        } else {
            console.log("Invalid choice. Please try again.");
        }
    }
}

main();

File: note_taking_app.py
import os

def create_note():
    title = input("Enter the title of the note: ")
    content = input("Enter the content of the note: ")
    with open(f"{title}.txt", "w") as file:
        file.write(content)
    print("Note created successfully.")

def view_note():
    title = input("Enter the title of the note to view: ")
    if os.path.exists(f"{title}.txt"):
        with open(f"{title}.txt", "r") as file:
            content = file.read()
        print("Note content:")
        print(content)
    else:
        print("Note not found.")

def delete_note():
    title = input("Enter the title of the note to delete: ")
    if os.path.exists(f"{title}.txt"):
        os.remove(f"{title}.txt")
        print("Note deleted successfully.")
    else:
        print("Note not found.")

def main():
    while True:
        print("1. Create a note")
        print("2. View a note")
        print("3. Delete a note")
        print("4. Exit")
        choice = input("Enter your choice: ")
        if choice == "1":
            create_note()
        elif choice == "2":
            view_note()
        elif choice == "3":
            delete_note()
        elif choice == "4":
            break
        else:
            print("Invalid choice. Please try again.")

if __name__ == "__main__":
    main()
